Makefile

```diff
@@ -89,9 +89,10 @@ ko-local: ## Build container images locally using ko
 # generate Go protobuf code
 protos:
 	@echo "Generating go protobuf files"
+	go install tool github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2
 	@mkdir -p ${OPENAPI_OUT}
 	${DOCKER_RUN} -v ${PWD}:${MOUNT_POINT} ${SIGSTORE_PROTO_BUILDER} \
-		-I/opt/include -I/googleapis -I/protobuf-specs -I${MOUNT_POINT}/api/proto \
+		-I/opt/include -I/googleapis -I/protobuf-specs -I${MOUNT_POINT}/third_party/googleapis -I${MOUNT_POINT}/api/proto \
```

Comment:
I think these should probably come from the common docker image builder if we want to keep these up to date using the sigstore/protobuf-specs infra.

I don't think that has to block this, but we should file a bug and update protobuf-specs.

Reply:
Can you give a more concrete example on where I should pull these from? I'd rather go ahead and get this right in this PR if I can.